packages/2018-neighborhood-development/src/components/ExploreUrbanCampsiteSweeps/text.js

```diff
@@ -0,0 +1,3 @@
+export const contextualDesc = 'With an estimated average of 5.8 (0-22) people per campsite, and an average of 24.7 sweeps per week, up to 143 people are being displaced on average each week. This map locates all campsite sweeps from October 2016-April 2018 around the city as points; this visualizes every time campers are required to vacate the area that they are surviving within.';
+export const belowFoldOne = 'The data were geocoded by location as points and counted over all weeks that reports are available, to look at longitudinal patterns over the previous 18 months in Portland (see Source link to methods report). Larger areas (e.g. multiple blocks, natural areas) that were describe as as single sweep in the raw data were coded as a single point that estimated the location of the area (either via a centroid estimate by hand, or a corner of the polygon).';
+export const belowFoldTwo = 'Take a look at campsite sweeps with neighborhood boundaries to see which neighborhoods have the most campsite sweeps. Look by bridgeheads, along freeways (I-84 and I-205), as well as near natural areas (the I-205 Multi-use Path, Springwater Corridor and the Columbia Slough, for example). The five neighborhoods with the most sweeps are Buckman, Lloyd Center, Lents, Old Town/Chinatown, and Hosford-Abernethy.';
```

Comment:
If you use backtick template strings, these can be multiline, btw.

Reply:
definitely going to incorporate going forward, thanks!